### ISSUE TITLE
Missing different cameras on different exposures of same tile

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1631]
A curious pipelining case, impacting Fuji:
```
80617TILEID  NIGHT   EXPID BADCAMWORD
------ -------- ----- ----------
 80617 20201222 69400         a6
 80617 20201222 69401         a6
 80617 20201222 69402         a3
 80617 20201222 69403         a3
 80617 20201222 69404         a3
```
Petal 6 is missing for two exposures, and petal 3 is missing for three other exposures on the same tile on the same night.  In principle the joint standard star fit could fit all 10 petals because every petal is represented on some exposure, but the pipeline is currently tied to the first exposure and thus generates standard stars for everything except petal 6.  This causes flux calibration to fail for exposures 69402-69404 because they are expecting petal 6 to exist (and don't care about petal 3).

**Comment by sbailey** [comment]
This is related to PR #1402 when individual frames are missing, but that didn't solve this case.

**Comment by djschlegel** [comment]
Conditions were poor that night.  This SV1 BGS+MWS tile 80617 collected 37 sec effective exposure time that night, compared to 24 min on the other four nights it was observed.  I suggest you mark those 5 exposures as bad and move on.

**Comment by sbailey** [comment]
@djschlegel thanks for checking effective times.  For the purposes of Fuji I like the solution of marking those 5 exposures as bad and moving on.

@akremin please take care of updating the exposure tables and purging the processing tables and matching preproc and exposures/ directories so that it will disappear from the dashboard as a problem.  When that is done please remove this from the Fuji project but let's keep the ticket open since this is a case that in general we should be able to support and it may come up again in the future.

**Comment by akremin** [comment]
69400-69404 are now marked to be ignore and I have removed the `preproc/ `and `exposures/` files for the 5 exposures.

**Comment by sbailey** [comment]
I think the general case was solved by PR #1658.  Closing this ticket.
